backend/src/main/java/de/iav/backend/controller/UserController.java

```diff
@@ -1,62 +1,85 @@
 package de.iav.backend.controller;
 
-import de.iav.backend.model.User;
-import de.iav.backend.model.UserWithoutUserDetails;
+import de.iav.backend.model.UserResponseDTO;
+
+import de.iav.backend.security.NewAppUser;
 import de.iav.backend.service.UserService;
+import jakarta.validation.Valid;
+import org.springframework.http.HttpStatus;
 import org.springframework.http.ResponseEntity;
 import org.springframework.web.bind.annotation.*;
 
 import java.net.URI;
-import java.util.List;
-import java.util.Optional;
+import java.util.*;
 
 @RestController
-@RequestMapping("/api/toastMasterManager/usersData")
+@RequestMapping("/api/toastMasterManager/users")
```

Comment:
Konvention ist folgende:

```suggestion
@RequestMapping("/api/toast-master-manager/users")
```

Reply:
Umgesetzt